engine/time/src/main/java/io/deephaven/time/calendar/BusinessCalendar.java

```diff
@@ -165,6 +165,7 @@ private YearData getYearData(final int year) {
 
     // region Getters
 
+    // TODO: should there be a string equivalent?
```

Comment:
I don't think there needs to be one

Reply:
I agree; people can convert these to strings on their own, or stick with java.time types (as we'd generally recommend).

I don't strongly oppose a `firstValidDateStr()`; I just wouldn't add it proactively. We can always add it later if people ask for it.